### ISSUE TITLE
Yaml issues in documentation

### ISSUE THREAD

**Issue by eloo** [promregator/promregator#68]
Hi,
im at the moment trying the targetAuth configuration and found an issue in the documentation.
https://github.com/promregator/promregator/blob/master/docs/outbound-authentication.md
the Yaml example there is invalid
```
 targetAuthenticators:
   - id: targetAuthenticator1
     type: basic
       username: targetUsername
       password: targetPassword

   - id: targetAuthenticatorNull
     type: null
```

the indention of username and password is wrong.. i guess it should be aligned with id and type?

thx

**Comment by eloo** [comment]
a looks like it should be 
```
 targetAuthenticators:
   - id: targetAuthenticator1
     type: basic
     basic:
       username: targetUsername
       password: targetPassword

   - id: targetAuthenticatorNull
     type: null
```

**Comment by eaglerainbow** [comment]
You've got a point there...
Should be fixed 5bfcbfa0ce4c98b660253885d895953a7f192103.
I cross-checked with a global search and couldn't find any further occurrence of this bug. 
In case you still find one, please raise your voice here!
